fastlane/metadata/de-DE/description.txt

```diff
@@ -30,6 +30,7 @@ ORGANISIEREN UND SUCHEN
 
 Datenschutzerklärung: https://automattic.com/privacy/
 Geschäftsbedingungen: https://simplenote.com/terms/
+California users privacy notice: https://wp.me/Pe4R-d/#california-consumer-privacy-act-ccpa
```

Comment:
I touched this and all the other ones manually.

I'm not sure whether it's appropriate to bypass the translation process? I thought it would be handy to have the links in already given this an hotfix.

Reply:
I think this only needs to be present in `en-US`?

But it's probably not hurting anything for it to be there – as long as it goes for translation. I guess the other question is do we have localized versions of that content at the URL? If it's only even in English, this probably doesn't help much

Reply:
My rationale for adding it to other locales too was that a user in California might not necessarily have their app in English. Although I do see your point and I think it's fair.

"What's the point of having it in `<language>` if it's not translate in `<language>`" is a valid question, too. 🤔 I wonder if there's a rule about this in CCPA?

The privacy policy in the Automattic page is translated only in a few languages right now. Also, while a user in California might have their phone in a language other than English, they should be able to _read_ English.

All in all, maybe I was bit rushed in adding those. @belcherj @jorgeatorres what do you think? I can easily open a PR against `develop` to remove them.